src/main/java/tutorspet/logic/util/AttendanceRecordListUtil.java

```diff
@@ -23,6 +25,55 @@
  */
 public class AttendanceRecordListUtil {
 
+    /**
+     * Returns an {@code AttendanceRecordList} where the {@code studentToRemove} has been removed.
+     */
+    public static AttendanceRecordList removeStudentFromAttendanceRecordList(
+            AttendanceRecordList targetAttendanceRecordList, Student studentToRemove) {
+        requireAllNonNull(targetAttendanceRecordList, studentToRemove);
+
+        List<AttendanceRecord> attendanceRecords = targetAttendanceRecordList.getAttendanceRecordList();
+
+        List<AttendanceRecord> updatedAttendanceRecords =
+                attendanceRecords.stream().map(attendanceRecord ->
+                        removeStudentFromAttendanceRecord(attendanceRecord, studentToRemove))
+                        .collect(Collectors.toUnmodifiableList());
+
+        return new AttendanceRecordList(updatedAttendanceRecords);
+    }
+
+    /**
+     * Returns an {@code AttendanceRecordList} where all {@code Student}s has been removed.
+     */
+    public static AttendanceRecordList removeAllStudentsFromAttendanceRecordList(
+            AttendanceRecordList targetAttendanceRecordList) {
+        requireAllNonNull(targetAttendanceRecordList);
+
+        List<AttendanceRecord> attendanceRecords = targetAttendanceRecordList.getAttendanceRecordList();
+
+        List<AttendanceRecord> updatedAttendanceRecords =
+                attendanceRecords.stream().map(attendanceRecord -> new AttendanceRecord())
+                        .collect(Collectors.toUnmodifiableList());
+
+        return new AttendanceRecordList(updatedAttendanceRecords);
+    }
+
+    /**
+     * Returns an {@code AttendanceRecord} where the {@code studentToRemove} has been removed.
+     */
+    private static AttendanceRecord removeStudentFromAttendanceRecord(
```

Comment:
Would it be possible to move private methods to the bottom of the class so that it can be grouped with the rest of the private methods?
```suggestion
    /**
     * Returns an {@code AttendanceRecord} where the {@code Attendance} of the {@code studentToRemove} has been removed.
     * If the {@code studentToRemove} does not exist in the {@code attendanceRecord}, the {@code attendanceRecord} is returned unchanged.
     */
    private static AttendanceRecord removeStudentFromAttendanceRecord(
```